tests/functional/disaster-recovery/regional-dr/conftest.py

```
@@ -658,3 +659,57 @@ def mirror_rdr_images(request):
+    try:
+        for cluster_name in dr_cluster_relations[0]:
+            cluster_index = config.get_cluster_index_by_name(cluster_name)
+            config.switch_ctx(cluster_index)
+            hco_obj = OCP(
+                kind=constants.HYPERCONVERGED,
+                namespace=constants.CNV_NAMESPACE,
+            )
+            if not hco_obj.is_exist(resource_name=constants.KUBEVIRT_HYPERCONVERGED):
+                log.info(
+                    f"HyperConverged resource '{constants.KUBEVIRT_HYPERCONVERGED}' not found "
+                    f"on cluster '{cluster_name}' — skipping test"
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)